templates/includes/sidebar.html

```diff
@@ -1,4 +1,20 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>
+      <a href="{% url 'taxi:driver-detail' user.id %}">Driver: {{ user.first_name }}</a>
```

Comment:
Ensure that the URL pattern 'taxi:driver-detail' accepts the user ID as a parameter. If the URL pattern does not match, it will result in a NoReverseMatch error.